lib/asciidoctor-pdf/index_catalog.rb

```diff
@@ -1,6 +1,6 @@
 module Asciidoctor; module Pdf
   class IndexCatalog
-    LeadingAlphaRx = /^[[:alpha:]]/
+    LeadingAlphaRx = /^[[:alpha:]]/u
```

Comment:
I would prefer that you use the following: `\p{Alpha}`. This is consistent with what we have done in core and elsewhere.

Reply:
When it should be `\p{L}` not `\p{Alpha}` for unicode support as far as i know

Reply:
\p{Alpha} in Ruby includes all Unicode. This is documented specifically in the language documentation.

Reply:
rechecked, \p{Alhpa} is fine with cyrillic in ruby.

Reply:
In fact, `[[:alpha:]]` on Ruby is already Unicode aware.

```
/[[:alpha:]]/.match? 'é'
# => true
```

The only thing that doesn't support unicode are character ranges:

```
/[a-z]/.match? 'é'
```

Reply:
But I still like the idea of using `\p{Alpha}` instead of `[[:alpha:]]` because it's more clear that it's modern and thus Unicode aware. (Very old versions of Ruby in fact didn't support Unicode in `[[:alpha:]]`).

Reply:
Can you give me sample words that your testing (until we have a test suite).

Reply:
here is cyrillic words i'm using together that similar latin ones

Абв
Abc
Бвг
Модульный
Modular

Reply:
yes it seems that there is no issue with current master  :(
i rechecked, i had v1.5.0 Alpha 16 locally installed, instead of regexp it uses ord with character range.

Reply:
Excellent. We'll proceed with this regardless for consistency.